pallets/runtime/tests/src/contract_test.rs

```diff
@@ -625,6 +635,7 @@ fn check_transaction_rollback_functionality_for_instantiation() {
     ExtBuilder::default()
         .network_fee_share(Perbill::from_percent(30))
         .set_protocol_base_fees(protocol_fee)
+        .set_contracts_put_code(true)
         .build()
         .execute_with(|| {
```

Comment:
This is repeated over and over again, including the `let`s before it. Let's make a HoF for that & refactor.